### PR TITLE
[main] Add Konflux configurations

### PR DESCRIPTION
**Pull request by serverless-qe**
Add Konflux components and pipelines

### CODE REVIEW

File: .tekton/serverless-bundle-135-pull-request.yaml [hunk 1]
```diff
@@ -7,14 +7,13 @@ metadata:
...
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "main"  && ( files.all.exists(x, x.matches('^olm-catalog/serverless-operator/')) || files.all.exists(x, x.matches('^.tekton/')) )
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && ( files.all.exists(x, x.matches('^olm-catalog/serverless-operator/')) || files.all.exists(x, x.matches('^.tekton/')) )
   creationTimestamp: null
   labels:
     appstudio.openshift.io/application: serverless-operator-135
-    appstudio.openshift.io/component: serverless-bundle-main
+    appstudio.openshift.io/component: serverless-bundle-135
```
Comment (creydr) [inline .tekton/serverless-bundle-135-pull-request.yaml:14]: @pierDipi this is strange, that this was working, even we renamed the component a couple of weeks ago (https://github.com/openshift-knative/hack/tree/main/.konflux/applications/serverless-operator-135/components and https://github.com/openshift-knative/hack/pull/334/commits/d15fad9f310aec6c0d34d630fd0fde6e40b2091d)

:thinking:

File: .tekton/serverless-index-135-fbc-413-pull-request.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - name: hermetic
+      value: "true"
+    - name: image-expires-after
+      value: 5d
+    - name: output-image
+      value: quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135-fbc-413/serverless-index-135-fbc-413:on-pr-{{revision}}
+    - name: revision
+      value: '{{revision}}'
+  pipelineRef:
+    name: docker-build
```
Comment (creydr) [inline .tekton/serverless-index-135-fbc-413-pull-request.yaml:39]: /hold
wrong pipeline

Comment (creydr) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml:39]: https://github.com/openshift-knative/hack/pull/341

File: .tekton/serverless-bundle-135-pull-request.yaml (hunk 1 above)
Comment (pierDipi) [reply .tekton/serverless-bundle-135-pull-request.yaml:14]: maybe that's why we didn't see the EC run here?

File: .tekton/serverless-index-135-fbc-413-pull-request.yaml (hunk 2 above)
Comment (creydr) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml]: /unhold

File: .tekton/serverless-source-image-135-pull-request.yaml
Comment (creydr) [inline .tekton/serverless-source-image-135-pull-request.yaml:1]: /hold
we don't need the source image (https://github.com/openshift-knative/hack/pull/343)

Comment (creydr) [reply .tekton/serverless-source-image-135-pull-request.yaml:1]: /unhold

File: .tekton/serverless-index-135-fbc-413-pull-request.yaml [hunk 3]
```diff
@@ -7,32 +7,32 @@ metadata:
...
+    appstudio.openshift.io/component: serverless-index-135-fbc-413
     pipelines.appstudio.openshift.io/type: build
-  name: serverless-index-main-on-pull-request
+  name: serverless-index-135-fbc-413-on-pull-request
   namespace: ocp-serverless-tenant
 spec:
   params:
     - name: dockerfile
-      value: olm-catalog/serverless-operator-index/Dockerfile
+      value: Dockerfile
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml:21]: I saw your point on the initial PR about this but now wondering how does Tekton knows about the component context

[the remark this answers is not in the document]
Comment (creydr) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml:21]: Thats true. The update here in the pipeline file seems wrong

[the remark this answers is not in the document]
Comment (creydr) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml:21]: need to set the context param too
```
  - default: .
    description: Path to the source code of an application's component from where
      to build image.
    name: path-context
    type: string
  - default: Dockerfile
    description: Path to the Dockerfile inside the context specified by parameter
      path-context
    name: dockerfile
    type: string
```

[the remark this answers is not in the document]
Comment (creydr) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml:21]: /hold

[the remark this answers is not in the document]
Comment (creydr) [reply .tekton/serverless-index-135-fbc-413-pull-request.yaml:21]: https://github.com/openshift-knative/hack/pull/350

**Pull request merged**